lib/iris/tests/unit/analysis/scipy_interpolate/test__RegularGridInterpolator.py

```diff
@@ -0,0 +1,80 @@
+# (C) British Crown Copyright 2015, Met Office
+#
+# This file is part of Iris.
+#
+# Iris is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Iris is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Iris.  If not, see <http://www.gnu.org/licenses/>.
+"""Unit tests for the
+:func:`iris.analysis._scipy_interpolate._RegularGridInterpolator` class."""
+
+from __future__ import (absolute_import, division, print_function)
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests
+
+import mock
+
+import numpy as np
+import iris
+
+from iris.analysis._scipy_interpolate import _RegularGridInterpolator
+from scipy.sparse.csr import csr_matrix
+import iris.tests.stock as stock
+
+
+class Test(tests.IrisTest):
+    def setUp(self):
+        '''Load a source cube, then generate an interpolator instance and
```

Comment:
This docstring isn't formatted correctly according to the style guidelines. Personally I'd just have it as 2 lines with a `#` comment in front of each, although it probably doesn't matter for the `setUp` fixture (if you put a docstring on an actual test method nose will print the docstring instead of the location of the test in its output, which is way less useful).